### ISSUE TITLE
great looking CMS

### ISSUE THREAD

**Issue by istvano** [directus/docs#15]
Hi Guys,

I have a question and I am not sure if this is the right place to ask it.

We have "reference book" style technical information document.  The information is intended to be found quickly when needed. 
Reference works are usually referred to for particular pieces of information, rather than read beginning to end.
This book is intended to be consulted for information on specific matters.

The source of the current book exists as Word and InDesign document.
Currently it is distributed as a ePub but it can be printed as needed.

We would like to convert this offering to the following:

- the book's content needs to be in a content repository where content is disconnected from it's format
- the content needs to be very easily editable using a GUI
- the content will be tagged and categorized by the editors
- the content needs to be versioned with the possiblity to track who changed what
- possibility to do multi format publishing ( Microsoft Word, PDF, Website, Mobile app, ePub) - pluggable output
- very robust search function ( Elasticsearch or SOLR or similar )
- replace the images with Videos and be able to output still images for the PDF and Streaming Video for on-line or mobile publicatinos
- the ability to read the data via APIs 
- perhaps export it to XML to be further processed etc.

would this CMS be the right approach for me ?

**Comment by benhaynes** [comment]
Thanks @istvano – yes, Directus would be an excellent solution for that. You would design your database exactly how you want (with content agnostic of formatting) and Directus would provide an intuitive way to manage/edit/create that content. While Directus has search built in to the admin panel, you'll likely be pulling in content to your app, which can use Elastic search or any other software... Directus doesn't limit this in any way.

Basically your data (content) would be clean, organized, and available through our API, SDK, or directly from the database. Any exporting, formatting,  or searching you do in your app is entirely up to you.

Hope this helps!

**Comment by istvano** [comment]
Hi, 

great thanks a lot
